lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js

```diff
@@ -47,10 +47,10 @@ class ExternalAnchorsUseRelNoopenerAudit extends Audit {
       });
     }
 
-    const pageHost = url.parse(artifacts.URL.finalUrl).host;
+    const pageHost = (new URL(artifacts.URL.finalUrl)).host;
```

Comment:
Ditto.  The comments are wrong, but we catch more by using host. IOW, if the user's app has the violation, we should flag it, no matter the port or protocol.